config/site_settings.yml

```diff
@@ -357,6 +357,10 @@ basic:
     default: "arial"
     enum: "BaseFontSetting"
     refresh: true
+  publish_sitemaps:
```

Comment:
this name is not that coherent with the rest of the settings, maybe: 

`enable_sitemap` ? - technically there is one sitemap.xml and it refs other files.

Reply:
I wanted to avoid confusion as the plugin's setting has a similar name (`sitemap_enabled`). I'll rename it to `enable_sitemap` since it won't cause any conflicts during the transition.